python/tests/conftest.py

```diff
@@ -286,6 +314,8 @@ def object_store_factory(request):
     Designed to test all object stores and their simulations
     Doesn't support LMDB
     """
+    if request.param == "azure_no_ssl_store_factory" and "tests.stress" not in str(request.module):
```

Comment:
I don't think we should skip tests this way - if the stress tests need to use a different set of fixtures we should change them directly, rather than unintuitively skipping tests here.